pkg/splunk/enterprise/afwscheduler_test.go

```diff
@@ -3048,6 +3048,10 @@ func TestPremiumAppScopedPlaybook(t *testing.T) {
 		t.Errorf("runPlayBook should have returned error")
 	}
 
+	if ! strings.Contains(err.Error(),"premium scoped app package install failed") {
```

Comment:
Nit: space between '!' and the conditional statement

Reply:
done